modules/mo_pytorch/mo_extensions/front/pytorch/conv_ext.py

```diff
@@ -21,23 +21,21 @@
 from mo.ops.convolution import Convolution
 from mo.utils.error import Error
 
+from .common import get_pads
 
 class Conv2dFrontExtractor(FrontExtractorOp):
     op = 'Conv2d'
     enabled = True
 
     @classmethod
     def extract(cls, node):
-        # Extract pads attribute
-        pads = np.array(node.module.padding, dtype=np.int64).reshape(1, 2)
-        pads = np.repeat(pads, 2, axis=0)
-        final_pads = np.array([[0, 0], [0, 0], *pads], dtype=np.int64)
+        final_pads = get_pads(node.module.padding)
```

Comment:
It's better to accept nn.Moduleas input:
```python
final_pads = get_pads(node.module)
```
Internally, get `.padding` attribute